glitter/migrations/0003_reminder.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
```

Comment:
Something isn't quite right here.

The migrations for this should be in `glitter/reminders/migrations`.